phylanx/execution_tree/tiling_annotations.hpp

```diff
@@ -67,7 +67,8 @@ namespace phylanx { namespace execution_tree
     {
         static char const* get_span_name(std::size_t i)
         {
-            static const char* const names[] = {"columns", "rows", "pages"};
+            static const char* const names[] = {
+                "columns", "rows", "pages", "quats"};
```

Comment:
Should this be `"quads"` instead?

Reply:
I think we named it `quats` as it was the first dimension of quaternions ([here](https://github.com/STEllAR-GROUP/phylanx/blob/master/phylanx/util/serialization/blaze.hpp#L196)).